src/chains/ethereum/transaction/src/access-lists.ts

```diff
@@ -80,4 +80,65 @@ export class AccessLists {
       dataFeeEIP2930: dataFee
     };
   }
+
+  public static isValidAccessList(accessList: AccessList): boolean {
```

Comment:
Because isValidAccessList partial guarantees validity (it's checking the accesslist shape, and length of values, but not the actual values), it needs either:

- JSDoc comment explaining to what degree the validity is assured
- Should be something like `tryGetAccessList` that validates _and_ converts to `Data` types (then you can leverage the validation within `Data`)

Reply:
Implemented the second option!